packages/layerchart/src/routes/docs/examples/Beeswarm/+page.svelte

```diff
@@ -36,6 +38,7 @@
               y: yForce.y(context.height / 2),
               collide: collideForce.radius(r),
             }}
+            data={{ nodes }}
```

Comment:
Remove `<Chart data={usSenators}` (unable to reference specific line in comment)

Reply:
I don't think we can do that. the contents of the `children` snippet make use of its `context` argument.

Reply:
Awe, I missed we use the data to add the axis and also the x accessor.